src/prommis/leaching/leach_solids_properties.py

```diff
@@ -193,10 +193,16 @@ def conversion_eq(b, j):
             )
 
     def get_material_flow_terms(self, p, j):
-        return self.flow_mass * self.mass_frac_comp[j] / self.params.mw[j]
+        return units.convert(
+            self.flow_mass * self.mass_frac_comp[j] / self.params.mw[j],
+            to_units=units.mol / units.hour,
+        )
```

Comment:
Did you add this unit conversion because some test of unit consistency was failing?